test/drop_test/config.py

```diff
@@ -45,6 +45,9 @@ def __init__(self, size: int, id: str):
     "thisisaverylongfilenameusingonlylowercaselettersandnumbersanditcontainshugestringofnumbers01234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234561234567891234567891234567890123456789012345678901234567890123456.txt": TestFile(
         size=1 * 1024, id="p3cYEJWds51Uqb174hrbHxfE6ASz_ioErmh39jLsiWU"
     ),
+    "thisisaverylongfilenameusingonlylowercaselettersandnumbersanditcontainshugestringofnumbers01234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234561234567891234567891234567890123456789012345678901234567890123456/testfile.txt": TestFile(
```

Comment:
For such a testcase python's ability to repeat the string as `c*250` could improve it slightly

Reply:
and the testcases below